CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CovertInfiltrationInfo.uc

```diff
@@ -160,8 +157,7 @@ static function CreateInfo(optional XComGameState StartState)
 	
 	Info = XComGameState_CovertInfiltrationInfo(NewGameState.CreateNewStateObject(class'XComGameState_CovertInfiltrationInfo'));
 	Info.ModVersion = CURRENT_MOD_VERSION;
-	Info.InitExistingCampaign();
```

Comment:
`InitExistingCampaign()` still does stuff unrelated to the crew limit, and I don't believe it's called elsewhere

Reply:
Good catch